### ISSUE TITLE
TrackBarAccessibilityObject_Value_Set_GetReturnsExpected failing in CI

### ISSUE THREAD

**Issue by JeremyKuhne** [dotnet/winforms#3992]
https://dev.azure.com/dnceng/public/_build/results?buildId=828192&view=ms.vss-test-web.build-test-results-tab&runId=26395620&resultId=105201&paneView=debug

```
System.Windows.Forms.Tests.AccessibleObjects.TrackBarAccessibleObjectTests.TrackBarAccessibilityObject_Value_Set_GetReturnsExpected(value: \"100\", expected: 10, expectedValueString: \"100\", createControl: True)

Error message
Assert.Equal() Failure\r\nExpected: 100\r\nActual:   (null)


Stack trace
   at System.Windows.Forms.Tests.AccessibleObjects.TrackBarAccessibleObjectTests.TrackBarAccessibilityObject_Value_Set_GetReturnsExpected(String value, Int32 expected, String expectedValueString, Boolean createControl) in /_/src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/TrackBarAccessibleObjectTests.cs:line 86
```

**Comment by kant2002** [comment]
My guess that this issue appears because of following situation.

1. Accessibility object ask TrackBar set Value to 100
2. TrackBar send message using `User32.SendMessageW(this, (User32.WM)TBM.SETPOS, (nint)BOOL.TRUE, reflectedValue);`
3. Somehow event does not send. My guess that it is message blocked by UIPI. See https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-sendmessage#remarks

I experience similar problem with `SplitContainerAccessibleObject_GetPropertyValue_HasKeyboardFocus_ReturnsTrue_IfControlHasFocus` where combination of `SetFocus`/`GetFocus` essentially works for setting kbd focus. 

https://github.com/dotnet/winforms/issues/4906 seems to be same as `SplitContainerAccessibleObject_GetPropertyValue_HasKeyboardFocus_ReturnsTrue_IfControlHasFocus`.

My guess that issue not in the tests itself, or configuration of build machines, but rather how we execute tests using `WinFormFact` machinery.

**Comment by JeremyKuhne** [comment]
Closing until we see this again.
